data_expansion1/localized/de_patch/text/game/items.stringtable

```diff
@@ -1658,7 +1658,7 @@ Als sie am nächsten Tag erwachte, fand sie ihn am Feuer vor, wo er saß und ein
       <ID>2344</ID>
       <DefaultText>Dieser dunkelgrüne emaillierte Stahlbrustpanzer war das Eigentum von Ryona, einer Abenteurerin aus den Vailianischen Republiken. Sie fand ihre Berufung darin, mit Expeditionstrupps den Dyrwald zu erforschen. Unter dem Brustpanzer befindet sich ein kupferner Kettenpanzer als zusätzlicher Schutz.
 
-Ryona beendete ihre Laufbahn verfrüht, nachdem sie ihren Expeditionstrupp in der Weißmark an einen Drachen verriet, um ihr eigenes Leben zu retten.</DefaultText>
+Ryona beendete ihre Laufbahn verfrüht, nachdem sie ihren Expeditionstrupp in der Weißmark an einen Drachen verraten hatte, um ihr eigenes Leben zu retten.</DefaultText>

```

Comment:
Ryona ended her career early after giving up her expedition group to a dragon in the White March in exchange for her own life